src/app/match-driver/[id]/page.tsx

```diff
@@ -68,11 +69,16 @@ export default async function DriverDetailPage({ params }: { params: { id: strin
               <p className="lg:text-[2.4rem] lg:leading-[3.2rem] sm:text-[1.6rem] sm:leading-[2.6rem] font-bold text-black-400">
                 리뷰 ({reviewData.reviewCount})
               </p>
-              <ReviewChart data={reviewData.reviews} score={reviewData.score} reviewCount={reviewData.reviewCount} />
+              {reviewData.reviewCount > 0 ? (
+                <ReviewChart data={reviewData.reviews} score={reviewData.score} reviewCount={reviewData.reviewCount} />
+              ) : (
+                <div className="py-[8rem]">
+                  <Empty type="Driver" />
+                </div>
+              )}
```

Comment:
```suggestion
{reviewData.reviewCount ? (
                <ReviewChart data={reviewData.reviews} score={reviewData.score} reviewCount={reviewData.reviewCount} />
              ) : (
                <div className="py-[8rem]">
                  <Empty type="Driver" />
                </div>
              )}
```
이렇게 하는 것이 어떨까요? 0보다 크면 무조건 truthy이고, 0이라면 무조건 falsy라서, > 0 이라는 로직이 필요할까 하는 생각이 듭니다

Reply:
넵 알겠습니다 반영하겠습니다 !